Created data_schema.sql and added dns masquerading table

aloftus23 opened the pull request:
Created the data_schema.sql file and added the domain masquerading schema.

# <!-- Use the title to describe PR changes in the imperative mood --> #

## 🗣 Description ##

Created the data_schema.sql file, added a table for the dns masquerading data, and provided comments.
<!-- Describe the "what" of your changes in detail. -->
<!-- To avoid scope creep, limit changes to a single goal. -->

## 💭 Motivation and context ##
<!-- Why is this change required? -->
To show how we will be storing domain masquerading data.

resolves #34 (Domain Masquerading Data Schema)
<!-- What problem does this change solve? How did you solve it? -->
<!-- Mention any related issue(s) here using appropriate keywords such
<!-- as "closes" or "resolves" to auto-close them on merge. -->

## 🧪 Testing ##

<!-- How did you test your changes? How could someone else test this PR? -->
Passes all pre-commits and pytests. Will not affect other areas of the code.
<!-- Include details of your testing environment, and the tests you ran to -->
<!-- see how your change affects other areas of the code, etc. -->


## ✅ Checklist ##

<!-- Remove any of the following that do not apply. -->
<!-- Draft PRs should have one or more unchecked boxes. -->
<!-- If you're unsure about any of these, don't hesitate to ask. -->
<!-- We're here to help! -->

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [ ] _All_ future TODOs are captured in issues, which are referenced
      in code comments.
- [ ] All relevant type-of-change labels have been added.
- [ ] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [ ] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [ ] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [ ] Tests have been added and/or modified to cover the changes in this PR.
- [ ] All new and existing tests pass.

aloftus23 committed:
Added domain masquerading table

Created the data_schema.sql file and added the domain masquerading schema.

schmelz21 commented:
@aloftus23, for now this should get us started as we gather resources. I am going to do a separate PR for dark web when ready.. still working out scheme.

aloftus23 committed:
updated dnstwist table

Updated table

- added primary key
- added if not exists
- added foreign key to connect with domain table

aloftus23 commented on src/pe_reports/data/data_schema.sql:
```diff
@@ -0,0 +1,18 @@
+
+-- Domain Masquerading Table
+CREATE TABLE "DNSTwist" (
+    "domain-name" character varying(200),
```
@cduhn17 I actually made a new commit about an hour ago that you should check out. This change is referring to the commit yesterday/

aloftus23 committed:
Added org and domain tables

- Added organization and domain tables

- relations between tables

aloftus23 commented:
<img width="574" alt="Screen Shot 2021-08-26 at 3 46 51 PM" src="https://user-images.githubusercontent.com/79927030/131026300-8c6212bb-62ce-4707-bffc-96f416b40915.png">

schmelz21 committed:
Merge branch 'develop' into AL-dataSchema
